openqaoa/backends/simulators/qaoa_analytical_sim.py

```diff
@@ -0,0 +1,119 @@
+#   Copyright 2022 Entropica Labs
```

Comment:
Since this is a new file, maybe format it right away with https://github.com/psf/black ?

Reply:
Formatted it with black, thanks!